core/client/fs/src/main/java/alluxio/client/meta/MetaMasterConfigClient.java

```diff
@@ -75,4 +75,13 @@ void setPathConfiguration(AlluxioURI path, Map<PropertyKey, String> properties)
    * @param path the path
    */
   void removePathConfiguration(AlluxioURI path) throws IOException;
+
+  /**
+   * Update properties.
+   *
+   * @param propertiesMap the properties map to be updated
+   * @return the update properties status map
+   */
+  Map<String, Boolean> updateConfiguration(
```

Comment:
check my comments https://github.com/Alluxio/alluxio/pull/13514#discussion_r645950175 , will that work?
I want to prevent the confusion in the future when adding new methods in this class.

Reply:
I think the strict type-safe protocol can lead we have to keep use same version both client and master.

Reply:
I'm a bit confused here. This API is currently called by Alluxio CLI, not by Alluxio client (e.g., inside Presto or FUSE). 
Do you see the possibility that even the Alluxio shell is in a different version of Alluxio deployment in this case?
We can discuss this offline

Reply:
You can use `PropertyKey.getOrBuildCustom` to create `PropertyKey`

Reply:
Done